tests/unit/objects/test_merge_requests.py

```diff
@@ -93,6 +94,12 @@ def resp_get_merge_request_reviewers():
         yield rsps
 
 
+def test_list_related_issues(project, resp_list_merge_requests):
+    mrs = project.mergerequests.list()
+    assert isinstance(mrs[0], ProjectMergeRequest)
+    assert mrs[0].related_issues == mr_content["related_issues"]
```

Comment:
I'm guessing you'll want to call the method here:

```suggestion
    assert mrs[0].related_issues() == mr_content["related_issues"]
```